pyproject.toml

```diff
@@ -24,6 +24,7 @@ dependencies = [
   "anndata",
   "boltons",
   "braceexpand",
+  "crick",
```

Comment:
Could you please specify a version you're using? Otherwise I can get into my problem with crick again after I update cellarium-ml to the latest version..